04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,84 @@
 #write your code here
+
+def translate(input)
+    vowels = ["a", "e", "i", "o", "u"]
+    if !input.include?(" ")
```

Comment:
not entirely sure if there's a best or better way of splitting out the logic, but making sure you know you can play around with case/switch statements too!

https://www.geeksforgeeks.org/ruby-case-statement/